### DISCUSSION TITLE
Spanish language

### DISCUSSION THREAD

**Discussion by Bricobit** [valinet/ExplorerPatcher#3838] [Ideas]
Is there any plan to add Spanish language translation support to the ExplorerPatcher interface?

**Comment by pyrates999** [comment]
if you're able to do the translation, you can submit a pull request for it.

**Reply by Bricobit** [reply]
What are the files that contain the translations? I have searched for them but I can't find anything related.

**Reply by pyrates999** [reply]
looks like this file contains the english translations: `ep_gui/resources/lang/ep_gui.en-US.rc`

**Comment by Bricobit** [comment]
I have already made the fork and added the Spanish language and made the pull request but it seems that there are not enough people reviewing the pull requests.

**Reply by pyrates999** [reply]
You just need to wait for one of the developers to review it, that's all.

**Comment by Bricobit** [comment]
Ok thanks

**Reply by pyrates999** [reply]
You should leave this open until it is accepted.

**Comment by Bricobit** [comment]
ok
